apps/freeablo/engine/enginemain.cpp

```diff
@@ -179,7 +179,10 @@ namespace Engine
                     state->mCursorEmpty = false;
                 else
                     state->mCursorEmpty = true;
-                state->mCursorFrame = FAGui::cursorFrame;
+                if (mWorld->getCurrentPlayer())
```

Comment:
@Predelnik This is the wip inv refactor, it's almost done but I didn't quite finish it today